app/controllers/users_controller.rb

```diff
@@ -29,4 +29,17 @@ def show
       @days_until_rota = @next_rota_date - Date.today
     end
   end
+
+  def contact_information
+    @email = params[:id]
+
+    AuditEvent.new(
+      email: current_user,
+      event: {
+        message: "Viewed #{@email}",
```

Comment:
```suggestion
        message: "Viewed #{@email}'s contact details from PagerDuty",
```

in case we ever expand this audit to other things?

Reply:
Resolved in 5cc82a1